### ISSUE TITLE
draw_networkx_edges breaks with latest matplotlib-3.3.0

### ISSUE THREAD

**Issue by dPys** [networkx/networkx#4082]
FYI 

```   
 def draw_networkx_edges(G, pos,
                            edgelist=None,
                            width=1.0,
                            edge_color='k',
                            style='solid',
                            alpha=1.0,
                            arrowstyle='-|>',
                            arrowsize=10,
                            edge_cmap=None,
                            edge_vmin=None,
                            edge_vmax=None,
                            ax=None,
                            arrows=True,
                            label=None,
                            node_size=300,
                            nodelist=None,
                            node_shape="o",
                            connectionstyle=None,
                            **kwds):
        """Draw the edges of the graph G.
    
        This draws only the edges of the graph G.
    
        Parameters
        ----------
        G : graph
           A networkx graph
    
        pos : dictionary
           A dictionary with nodes as keys and positions as values.
           Positions should be sequences of length 2.
    
        edgelist : collection of edge tuples
           Draw only specified edges(default=G.edges())
    
        width : float, or array of floats
           Line width of edges (default=1.0)
    
        edge_color : color string, or array of floats
           Edge color. Can be a single color format string (default='r'),
           or a sequence of colors with the same length as edgelist.
           If numeric values are specified they will be mapped to
           colors using the edge_cmap and edge_vmin,edge_vmax parameters.
    
        style : string
           Edge line style (default='solid') (solid|dashed|dotted,dashdot)
    
        alpha : float
           The edge transparency (default=1.0)
    
        edge_ cmap : Matplotlib colormap
           Colormap for mapping intensities of edges (default=None)
    
        edge_vmin,edge_vmax : floats
           Minimum and maximum for edge colormap scaling (default=None)
    
        ax : Matplotlib Axes object, optional
           Draw the graph in the specified Matplotlib axes.
    
        arrows : bool, optional (default=True)
           For directed graphs, if True draw arrowheads.
           Note: Arrows will be the same color as edges.
    
        arrowstyle : str, optional (default='-|>')
           For directed graphs, choose the style of the arrow heads.
           See :py:class: `matplotlib.patches.ArrowStyle` for more
           options.
    
        arrowsize : int, optional (default=10)
           For directed graphs, choose the size of the arrow head head's length and
           width. See :py:class: `matplotlib.patches.FancyArrowPatch` for attribute
           `mutation_scale` for more info.
    
        connectionstyle : str, optional (default=None)
           Pass the connectionstyle parameter to create curved arc of rounding
           radius rad. For example, connectionstyle='arc3,rad=0.2'.
           See :py:class: `matplotlib.patches.ConnectionStyle` and
           :py:class: `matplotlib.patches.FancyArrowPatch` for more info.
    
        label : [None| string]
           Label for legend
    
        Returns
        -------
        matplotlib.collection.LineCollection
            `LineCollection` of the edges
    
        list of matplotlib.patches.FancyArrowPatch
            `FancyArrowPatch` instances of the directed edges
    
        Depending whether the drawing includes arrows or not.
    
        Notes
        -----
        For directed graphs, arrows are drawn at the head end.  Arrows can be
        turned off with keyword arrows=False. Be sure to include `node_size` as a
        keyword argument; arrows are drawn considering the size of nodes.
    
        Examples
        --------
        >>> G = nx.dodecahedral_graph()
        >>> edges = nx.draw_networkx_edges(G, pos=nx.spring_layout(G))
    
        >>> G = nx.DiGraph()
        >>> G.add_edges_from([(1, 2), (1, 3), (2, 3)])
        >>> arcs = nx.draw_networkx_edges(G, pos=nx.spring_layout(G))
        >>> alphas = [0.3, 0.4, 0.5]
        >>> for i, arc in enumerate(arcs):  # change alpha values of arcs
        ...     arc.set_alpha(alphas[i])
    
        Also see the NetworkX drawing examples at
        https://networkx.github.io/documentation/latest/auto_examples/index.html
    
        See Also
        --------
        draw()
        draw_networkx()
        draw_networkx_nodes()
        draw_networkx_labels()
        draw_networkx_edge_labels()
        """
        try:
            import matplotlib
            import matplotlib.pyplot as plt
            import matplotlib.cbook as cb
            from matplotlib.colors import colorConverter, Colormap, Normalize
            from matplotlib.collections import LineCollection
            from matplotlib.patches import FancyArrowPatch
            import numpy as np
        except ImportError:
            raise ImportError("Matplotlib required for draw()")
        except RuntimeError:
            print("Matplotlib unable to open display")
            raise
    
        if ax is None:
            ax = plt.gca()
    
        if edgelist is None:
            edgelist = list(G.edges())
    
        if not edgelist or len(edgelist) == 0:  # no edges!
            return None
    
        if nodelist is None:
            nodelist = list(G.nodes())
    
        # set edge positions
        edge_pos = np.asarray([(pos[e[0]], pos[e[1]]) for e in edgelist])
    
>       if not cb.iterable(width):
E       AttributeError: module 'matplotlib.cbook' has no attribute 'iterable'

/opt/circleci/.pyenv/versions/3.6.5/lib/python3.6/site-packages/networkx/drawing/nx_pylab.py:579: AttributeError
```

cbook.iterable is now deprecated. Should just be a quick swap out with np.iterable?

**Comment by rossbar** [comment]
Thanks for the report - the `np.iterable` approach seems like a viable solution at first glance. Please feel free to submit a PR with any suggested changes!

**Comment by dPys** [comment]
Thanks @rossbar , but it actually appears that this issue was already caught and addressed in #3458  a few weeks ago. Heads up though that anyone working with nx drawing utilities pre-version 2.4 will hit this error if they upgraded their mpl. 

Closing this!
